frontend/packages/integration-tests-cypress/tests/app/template.cy.ts

```diff
@@ -0,0 +1,31 @@
+import { catalog } from '../../views/catalogs';
+import { guidedTour } from '../../views/guided-tour';
+
+describe('template feature', () => {
+  before(() => {
+    cy.log('create template:');
+    cy.exec(`oc create -f ./fixtures/httpd-example-template.yaml -n openshift`).then((result) => {
+      expect(result.stdout).to.include('created');
+    });
+    cy.login();
+    guidedTour.close();
+  });
+
+  after(() => {
+    cy.exec('oc delete template httpd-example-test -n openshift');
+  });
+  it('Allow custom icon using template annotation', () => {
+    cy.visit('/catalog/ns/default');
```

Comment:
it would be better if we can navigate using `cy.clickNavLink(['Ecosystem', 'Software Catalog']);`